content/practices/pre-ipm/index.md

```diff
@@ -0,0 +1,84 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: true
```

Comment:
```suggestion
draft: false
```

Reply:
Please set to false to see the preview. I'm going to update the template to remove this, as it did not function as I expected.